src/overlays/actors/ovl_Obj_Ocarinalift/z_obj_ocarinalift.h

```diff
@@ -25,8 +25,8 @@ typedef struct ObjOcarinalift {
 } ObjOcarinalift; // size = 0x178
 
 typedef enum ObjOcarinaliftParams {
-    /* 0x0 */ OBJOCARINALIFT_PARAMSC_0,
-    /* 0x1 */ OBJOCARINALIFT_PARAMSC_1
+    /* 0 */ OBJOCARINALIFT_PARAMSC_0,
+    /* 1 */ OBJOCARINALIFT_PARAMSC_1
 } ObjOcarinaliftParams;
```

Comment:
```suggestion
} ObjOcarinaliftParam;
```
Can you remove the `s` from the other parts as well?